diff -urN darkice-0.7.org/configure.in darkice-0.7/configure.in
--- darkice-0.7.org/configure.in	Sun Oct 21 00:11:21 2001
+++ darkice-0.7/configure.in	Sun Oct 21 00:13:41 2001
@@ -40,7 +40,7 @@
 
 if test "x${USE_LAME}" = "xyes" ; then
     AC_MSG_CHECKING( [for lame library at ${CONFIG_LAME_PREFIX}] )
-    LA_SEARCH_LIB( LAME_LIB_LOC, LAME_INC_LOC, libmp3lame.a, lame/lame.h,
+    LA_SEARCH_LIB( LAME_LIB_LOC, LAME_INC_LOC, libmp3lame.so, lame/lame.h,
                    ${CONFIG_LAME_PREFIX})
     if test "x${LAME_LIB_LOC}" != "x" ; then
         AC_DEFINE( HAVE_LAME_LIB, 1, [build with lame library] )
@@ -72,12 +72,12 @@
 
 if test "x${USE_VORBIS}" = "xyes" ; then
     AC_MSG_CHECKING( [for vorbis libraries at ${CONFIG_VORBIS_PREFIX}] )
-    LA_SEARCH_LIB( OGG_LIB_LOC, OGG_INC_LOC, libogg.a, ogg/ogg.h,
+    LA_SEARCH_LIB( OGG_LIB_LOC, OGG_INC_LOC, libogg.so, ogg/ogg.h,
                    ${CONFIG_VORBIS_PREFIX})
-    LA_SEARCH_LIB( VORBIS_LIB_LOC, VORBIS_INC_LOC, libvorbis.a, vorbis/codec.h,
+    LA_SEARCH_LIB( VORBIS_LIB_LOC, VORBIS_INC_LOC, libvorbis.so, vorbis/codec.h,
                    ${CONFIG_VORBIS_PREFIX})
     LA_SEARCH_LIB( VORBISENC_LIB_LOC, VORBISENC_INC_LOC,
-                   libvorbisenc.a, vorbis/vorbisenc.h,
+                   libvorbisenc.so, vorbis/vorbisenc.h,
                    ${CONFIG_VORBIS_PREFIX})
 
     if test "x${OGG_LIB_LOC}" != "x" -a \
